src/lib/RenderString.js

```diff
@@ -0,0 +1,21 @@
+import * as AllIcons from '@lana/b2c-mapp-ui-assets/dist/index';
+
+const RenderString = {
+  props: {
+    string: {
+      required: true,
+      type: String,
+    },
+  },
+  render(createElement) {
+    const render = {
+      components: {
+        ...AllIcons,
+      },
+      template: `<div>${this.string}</div>`,
+    };
+    return createElement(render);
+  },
+};
+
+export default RenderString;
```

Comment:
This file should be named after camelCase standard

Reply:
changed